data/successors/successor side missions.txt

```diff
@@ -0,0 +1,488 @@
+			choice
+				`	"What caused the collapse?"`
+					goto collapse
+				`	"How large was their empire?"`
+					goto empire
+				`	"What was their technology like?"`
+					goto tech
+				`	"I have no more questions."`
+					launch
+			label collapse
+			`	The researcher sighs. "We do not know; or better put, the fullness of the truth yet remains obscured. The flesh of space itself was rent, distorted; the effects of this remain today: the holes embedded in the skies, the dying-out of light and even of the hyperlanes. It was a warring time; the Houses turned upon each other or they fled, and space itself was set alight with the blinks of shots and drives---"`
+			`	The researcher stops for a moment, composing themself.`
+			`	"I digress and do apologize. T'is challenging to separate the truth from fiction in this regard. Our Predecessors wielded power far beyond our modern imitations, the sort that sows the seeds of peace or chaos deep within the fabric of the world, the sort of things that poke and prod at places that must remain untouched. 'What caused the collapse?' you ask? Hubris, methinks."`
+			choice
+				`	"How large was their empire?"`
+					goto empire
+				`	"What was their technology like?"`
+					goto tech
+				`	"I have no more questions."`
+					launch
+			label empire
+			`	"The translation device thou wears upon thy person speaks volumes to its scale. Sieasej tells me that at that empire's peak, our scholars had even laid eyes upon thy own world, far to the north! These dead systems that remain where once the core of their empire stood are but a small fraction of what was there. Some, it's sure, now belong to other races, and others, it would seem, have been obscured. We know not how nor why, only that the starcharts of our Predecessors do not mirror those we see today."`
+			choice
+				`	"What caused the collapse?"`
+					goto collapse
+				`	"What was their technology like?"`
+					goto tech
+				`	"I have no more questions."`
+					launch
+			label tech
+			`	"Glorious and beautiful; as much art as science or engineering. At their peak, the works of our Predecessors were envied and respected even by the Quarg, whose mastery knows little equal. Twere exotic in their natures and unique in applications; living metal grown in stars, subtle points to pierce space itself. We have descriptions, enough to make rudimentary imitations of their processes, but even if we had a perfect understanding of wha'twas, the space of our new systems seems unsuitable to manifest that mastery. We fail even to replicate the heirlooms kept by the High Houses; the materials and techniques they use are always out of reach."`
```

Comment:
That first part does feel somewhat awkward, though I'm not sure my suggested change makes it much better.
```suggestion
			`	"Glorious and beautiful; as much art as science or engineering. At their peak, the works of our Predecessors were envied and respected even by the Quarg, whose mastery only few equal. T'were exotic in their natures and unique in applications; living metal grown in stars, subtle points to pierce space itself. We have descriptions, enough to make rudimentary imitations of their processes, but even if we had a perfect understanding of wha'twas, the space of our new systems seems unsuitable to manifest that mastery. We fail even to replicate the heirlooms kept by the High Houses; the materials and techniques they use are always out of reach."`
```